### PR TITLE
games-engines/exult: version 1.6

### PR DESCRIPTION
**Pull request by winterheart**
Stable version 1.6, remove live ebuild.
Closes: https://bugs.gentoo.org/752429

### CODE REVIEW

File: games-engines/exult/exult-1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+EAPI=7
+
+DESCRIPTION="an Ultima 7 game engine that runs on modern operating systems"
+HOMEPAGE="http://exult.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/${PN}/exult-all-versions/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="alsa opengl +sdl2 fluidsynth timidity tools zlib"
```
Comment (juippis) [inline games-engines/exult/exult-1.6.ebuild:13]: Everything looks good and this builds fine without any notices, but can we please sort this IUSE line alphabetically to make it perfect?

File: games-engines/exult/exult-1.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		--enable-mods \
+		--with-desktopdir=/usr/share/applications \
+		--with-icondir=/usr/share/pixmaps \
+		--with-sdl=$(usex sdl2 sdl2 sdl12) \
+		$(use_enable alsa) \
+		$(use_enable fluidsynth) \
+		$(use_enable opengl) \
+		$(use_enable timidity timidity-midi) \
+		$(use_enable tools) \
+		$(use_enable zlib zip-support)
```
Comment (juippis) [inline games-engines/exult/exult-1.6.ebuild:44]: Actually this does **not** compile with `-zlib`, although the error doesn't say anything about zlib. Could you take a look? Patch it or make it unconditional dep.
```
gamedat.cc:93:6: error: no declaration matches 'void Game_window::restore_flex_files(IDataSource&, const char*)'
   93 | void Game_window::restore_flex_files(
      |      ^~~~~~~~~~~
gamedat.cc:93:6: note: no functions named 'void Game_window::restore_flex_files(IDataSource&, const char*)'
In file included from gamedat.cc:42:
gamewin.h:94:7: note: 'class Game_window' defined here
   94 | class Game_window {
      |       ^~~~~~~~~~~
gamedat.cc: In member function 'void Game_window::restore_gamedat(const char*)':
gamedat.cc:231:2: error: 'restore_flex_files' was not declared in this scope
  231 |  restore_flex_files(in, GAMEDAT);
      |  ^~~~~~~~~~~~~~~~~~
```

**Comment by winterheart** [comment]
@juippis done.

File: games-misc/exult-sound/exult-sound-0-r2.ebuild [hunk 3]
```diff
@@ -1,18 +1,17 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 DESCRIPTION="Sound data for games-engines/exult"
 HOMEPAGE="http://exult.sourceforge.net/"
-SRC_URI="mirror://sourceforge/exult/exult_audio.zip"
+SRC_URI="https://download.sourceforge/exult/exult-data/exult_audio.zip"
```
Comment (juippis) [inline games-misc/exult-sound/exult-sound-0-r2.ebuild:8]: Will update this after merging, rest looks good, thanks!

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**